.gitlab-ci.yml

```
@@ -0,0 +1,120 @@
+stages:
+    - test
+    - package
+    - deploy
+
+variables:
+    ARTIFACT_PATH: eva-accession-ws/target/eva-$ENVIRONMENT_NAME.war
+
+before_script:
+    - export DATETIME=$(date +%Y-%m-%dT%H-%M-%S)
+
+test:
+    stage: test
+    image: maven:3.6.1-jdk-8-alpine
+    services:
+        - mongo:3.2.17
+    script:
+        - mvn clean test -pl 'eva-accession-core,eva-accession-ws' -Deva.mongo.host.test=mongo
```

Judgement call: The non-Unix convention of maven command line switch is a bit confusing since -pl constitutes one switch. Could we expand to the more intuitive --projects?
```suggestion
        - mvn clean test --projects 'eva-accession-core,eva-accession-ws' -Deva.mongo.host.test=mongo
```